app/forms/bulk_action_form.rb

```diff
@@ -30,16 +30,21 @@ def sync
 
   delegate :action_type, :description, to: :model
 
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
   def csv_as_string
     csv_file = create_virtual_objects&.fetch(:csv_file) ||
                register_druids&.fetch(:csv_file) ||
-               import_tags&.fetch(:csv_file)
+               import_tags&.fetch(:csv_file) ||
+               set_catkeys_and_barcodes&.fetch(:csv_file)
```

Comment:
Is `:csv_file` an arg for catkeys & barcodes? If so, I'd've expected to see it in the permitted params in the controller (which currently only has catkeys/_option and barcodes/_option.